lib/dispatcher.rb

```diff
@@ -36,8 +36,10 @@ def deliver_new_proposal_emails(proposal)
   end
 
   def deliver_cancellation_emails(proposal)
-    proposal.approvers.each do |approver|
-      CommunicartMailer.cancellation_email(approver.email_address, proposal).deliver_later
+    proposal.approvals.each do |approval|
+      next if approval.pending?  # https://www.pivotaltracker.com/story/show/100733040
```

Comment:
Mind adding a comment of something like "don't notify people who haven't heard about the proposal yet"?